meta-refkit/classes/refkit-image.bbclass

```diff
@@ -174,6 +174,10 @@ IMAGE_INSTALL += "util-linux"
 export ALTERNATIVE_PRIORITY_SHADOW ?= "305"
 IMAGE_INSTALL += "shadow"
 
+# Install thermal management daemon by default for
+# handling platform thermal management.
+IMAGE_INSTALL += "thermald"
```

Comment:
With the boards we currently have hardware does automatic thermal throttling and there are no PWM fans...